pkgs/games/bugdom/default.nix

```
@@ -0,0 +1,43 @@
+{ lib, stdenv, fetchFromGitHub, SDL2, cmake, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "bugdom";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "jorio";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256:1371inw11rzfrxmc3v4gv5axp56bxjbcr0mhqm4x839401bfq5mf";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [
+    SDL2
+    cmake
+    makeWrapper
+  ];
```

Only SDL2 is needed at runtime.

```suggestion
  buildInputs = [
    SDL2
  ];
  
  nativeBuildInputs = [
    cmake
    makeWrapper
  ];
```